team_token.go

```diff
@@ -13,23 +13,36 @@ import (
 // Compile-time proof of interface implementation.
 var _ TeamTokens = (*teamTokens)(nil)
 
+const (
+	AuthenticationTokensPath = "authentication-tokens/%s"
```

Comment:
The linter complained that this needed to be made a constant. I didn't see a good common file to place this in and didn't really want to create a new file with just one constant, so I put it here. Open to feedback, though!